pkgs/top-level/all-packages.nix

```diff
@@ -3400,6 +3400,8 @@ in
 
   fcitx-configtool = callPackage ../tools/inputmethods/fcitx/fcitx-configtool.nix { };
 
+  chewing-editor = libsForQt5.callPackage ../applications/misc/chewing-editor { };
```

Comment:
I believe that we have a general recommendation that the entries in all-packages.nix should roughly be alphabetically sorted. (This recommendation does not seem to be followed through rigorously, perhaps someone more experienced can give an indication why.)

Reply:
Thanks for telling me that.
(I'm a newbie here, and it's the first time that I make a PR in NixOS/nixpkgs.
The second commit actually reflects the suggestions from some senior developers on Discord on `default.nix`, or the compilation couldn't have passed.)
But it seems that there are several rounds of "sorted sections" in `all-packages.nix`. Where should I put my code in?

Reply:
I still haven't figure out the order yet, but I have it inserted somewhere between ch- and cl- to make `all-packages.nix` closer to being alphabetically sorted.
Thank you for your advice.

Reply:
Alphabetical order doesn't really matter and it is hard to check/enforce, so it doesn't happen. Just give it your best effort. Don't put stuff at the bottom though, because that is just asking for a merge conflict.